requirements.txt

```diff
@@ -0,0 +1 @@
+git+git://github.com/david415/python-ftools.git#egg=ftools
```

Comment:
I think we should take care to not add a dependency on a random Github repo last updated in 2010.

Is python ftools available on pypi? Perhaps we should see if it can be released there?